WalletWasabi.Gui/Rpc/WasabiJsonRpcService.cs

```diff
@@ -243,9 +246,9 @@ public void SelectWallet(string walletName)
 		}
 
 		[JsonRpcMethod("stop")]
-		public async Task StopAsync()
+		public void Stop()
```

Comment:
@lontivero Terminate will exit the application with Environment.Exit and never returns. Is this OK? Also, I had to make this sync.

Reply:
This is slowed down from 2 sec to 10 for some reason. I will investigate more.

Reply:
RPC terminating itself so is not very healthy to run that synchronously as it could get into deadlock - I tested it and happened randomly. 
With the current version, I use await and start the TErmination on another thread.